src/paper/serializers.py

```diff
@@ -267,6 +268,11 @@ def get_user_vote(self, paper):
                     pass
         return vote
 
+    def get_unified_document(self, paper):
+        from researchhub_document.serializers.researchhub_unified_document_serializer import MinimalUnifiedDocumentSerializer
+        uni_doc = MinimalUnifiedDocumentSerializer(paper.unified_document)
+        return uni_doc.data
+
```

Comment:
I think using a dynamic serializer here is better than defining a new serializer.

Reply:
Yeah sounds reasonable